gnucash/import-export/aqb/dialog-ab.glade

```diff
@@ -634,21 +634,136 @@
               </packing>
             </child>
             <child>
-              <object class="GtkImage" id="optical_challenge">
-                <property name="name">optical_challenge</property>
+              <object class="GtkBox" id="flicker_hbox">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
+                <property name="spacing">7</property>
+                <child>
+                  <object class="GtkLabel">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="halign">end</property>
+                    <property name="label" translatable="yes">Bar_width</property>
+                    <property name="xalign">0.7</property>
+                    <property name="use_underline">True</property>
+                    <property name="mnemonic_widget">spin_barwidth</property>
+                    <attributes>
+                      <attribute name="gravity" value="west"/>
+                    </attributes>
+                  </object>
+                  <packing>
+                    <property name="expand">False</property>
+                    <property name="fill">True</property>
+                    <property name="position">0</property>
+                  </packing>
+                </child>
+                <child>
+                  <object class="GtkSpinButton" id="spin_barwidth">
+                    <property name="visible">True</property>
+                    <property name="can_focus">True</property>
```

Comment:
Example for translatable tooltip with translator comment:
`                    <property name="tooltip-text" translatable="yes" comments="TAN generator with flicker interface common in DE only">Adjust bar width …</property>`